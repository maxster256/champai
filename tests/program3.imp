DECLARE
    array(5:100);
IN
    READ array(5);
    WRITE array(5);
END
