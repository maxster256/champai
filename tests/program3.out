SUB A A
SUB A A
INC A
GET H
STORE H
SUB A A
INC A
LOAD H
PUT H
HALT